docs/guide/routing.md

```diff
@@ -240,6 +240,56 @@ app.Get("/:test<int>?", func(c fiber.Ctx) error {
 // Cannot GET /7.0
 ```
 
+**Custom Constraint Example**
+
+Custom constraints can be added to Fiber using the `app.RegisterCustomConstraint` method. Your constraints have to be compatible with the `CustomConstraint` interface.
```

Comment:
Could we add 1-2 sentences here explaining the benefits/use-cases of doing Custom Constraints.

Reply:
Will do